packages/collector/test/tracing/cloud/azure/blob/test.js

```diff
@@ -0,0 +1,444 @@
+/*
+ * (c) Copyright IBM Corp. 2021
+ * (c) Copyright Instana Inc. and contributors 2018
+ */
+
+'use strict';
+
+const { v4: uuid } = require('uuid');
+const expect = require('chai').expect;
+const semver = require('semver');
+const constants = require('@instana/core').tracing.constants;
+const supportedVersion = require('@instana/core').tracing.supportedVersion;
+const config = require('../../../../../../core/test/config');
+const {
+    verifyHttpRootEntry,
+    expectExactlyNMatching
+} = require('../../../../../../core/test/test_util');
+const ProcessControls = require('../../../../test_util/ProcessControls');
+const globalAgent = require('../../../../globalAgent');
+const { fail } = expect;
+
+const { createContainer, deleteContainer, miniNodeJsVer } = require('./util');
+const { BlobServiceClient } = require('@azure/storage-blob');
+const containerName = `test-${uuid()}`;
+const storageAccount = process.env.STORAGE_ACCOUNT;
+const accountKey = process.env.ACCOUNT_KEY;
+const endPoint = 'core.windows.net';
+const connStr =
+`DefaultEndpointsProtocol=https;AccountName=${storageAccount};AccountKey=${accountKey};EndpointSuffix=${endPoint}`;
+const blobServiceClient = BlobServiceClient.fromConnectionString(connStr);
+const containerClient = blobServiceClient.getContainerClient(containerName);
+
+let mochaSuiteFn = supportedVersion(process.versions.node) ? describe : describe.skip;
+mochaSuiteFn = semver.lt(process.versions.node, miniNodeJsVer) ? describe.skip : mochaSuiteFn;
+
+mochaSuiteFn('tracing/cloud/azure/blob', function () {
+    globalAgent.setUpCleanUpHooks();
+    const agentControls = globalAgent.instance;
+
+    const controls = new ProcessControls({
+        dirname: __dirname,
+        useGlobalAgent: true,
+        env: {
+            CONTAINER_NAME: containerName,
+            CONN_STR: connStr,
+            STORAGE_ACC: storageAccount,
+            ACC_KEY: accountKey,
+            CONT_CLIENT: containerClient
+        }
+    });
+    ProcessControls.setUpHooks(controls);
+    if (!process.env.ACCOUNT_KEY) {
```

Comment:
Initially you are assigning process.env.ACCOUNT_KEY to a variable, the here you are accessing process.env again.

Reply:
used assigned variable